fdbserver/CommitProxyServer.actor.cpp

```diff
@@ -874,6 +876,104 @@ ACTOR Future<Void> preresolutionProcessing(CommitBatchContext* self) {
 	return Void();
 }
 
+namespace {
+ErrorOr<Optional<std::pair<TenantName, int64_t>>> getTenantDetails(ProxyCommitData* commitData,
+                                                                   Optional<int64_t> tenantId,
+                                                                   bool logOnFailure) {
```

Comment:
We don't seem to be using `logOnFailure` so we can probably just remove it.

Reply:
Current usage is `none` , however, idea was to make it a utility method useful in future similar scenario, can remove it if needed (we can add it later on demand as well) if you feel strong about it. 

Thanks

Reply:
I'd say keep the implementation of this function as simple as possible rather than trying to anticipate future needs. These things are easy enough to add later, and at that time we'll better understand what we want anyway.